plugins/InspectionGadgets/src/META-INF/InspectionGadgets.xml

```diff
@@ -285,6 +285,10 @@
                      key="new.string.buffer.with.char.argument.display.name" groupBundle="messages.InspectionsBundle"
                      groupKey="group.names.probable.bugs" enabledByDefault="true" level="WARNING"
                      implementationClass="com.siyeh.ig.bugs.NewStringBufferWithCharArgumentInspection"/>
+    <localInspection groupPath="Java" language="JAVA" shortName="MathRoundingWithIntArgument" bundle="messages.InspectionGadgetsBundle"
+                     key="math.rounding.with.int.argument.display.name" groupBundle="messages.InspectionsBundle"
+                     groupKey="group.names.probable.bugs" enabledByDefault="false" level="WARNING"
```

Comment:
In my opinion you should enable it by default. This seems like a useful inspection to me, which provides few but useful warnings.

In general it is basically not worth to implement an inspection if you are not going to enable it by default. It almost won't be used.

Reply:
Thank you! Enabled by default